src/makeInfinityAdder.js

```diff
@@ -7,6 +7,21 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  return function adder(add = false) {
+    sum += add;
```

Comment:
The function 'adder' could be an arrow function for more concise and readable code.